src/index.js

```diff
@@ -75,6 +75,56 @@ var _applyToPath = function(obj, path, val) {
   }
 };
 
+var doConcat = function(obj, args) {
```

Comment:
I think arr.concat is actually fine in this case. But helpers like contains and unique could start coming out of lodash. Definitely not a blocker just don't think we need to continue writing our own helpers in each repo that pretty much do the same thing